src/SFA.DAS.AssessorService.Web/ViewModels/Certificate/Private/CertificateUkprnViewModel.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Newtonsoft.Json;
+using SFA.DAS.AssessorService.Domain.JsonData;
+
+namespace SFA.DAS.AssessorService.Web.ViewModels.Certificate.Private
+{
+    public class CertificateUkprnViewModel : CertificateBaseViewModel, ICertificateViewModel
+    {
+        public string Ukprn { get; set; }
```

Comment:
Should Ukprn be a string? Normally it's a integer

Reply:
yep it's because I have a selector fro a dropdown list and dont want to display 0.